src/components/menu/Menu.tsx

```diff
@@ -83,7 +83,7 @@ const Menu: GenericComponent<MenuProps> = ({
     [uiUtilities['box-shadow-200']]: position !== POSITION.STATIC,
   });
 
-  const handleDocumentClick = (event: SyntheticEvent) => {
+  const handleDocumentClick = (event: Event) => {
```

Comment:
Q: could it be MouseEvent?

Reply:
No, because `DocumentEventMap` is using type `Event`